demo/helper-scripts/demo-pod-with-discovery.json.parsec.template

```diff
@@ -0,0 +1,25 @@
+{
+    "apiVersion": "v1",
+    "kind": "Pod",
+    "metadata": {
+        "name": "demo-parsec-NUM"
+    },
+    "spec": {
+        "containers": [
+            {
+                "image": "intelsdi/node-feature-discovery-APP",
```

Comment:
If you want, we can create a separate issue to do that. It doesn't need to block merging the rest of the demo resources.